include/bluetooth/services/hids_common.h

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Can't this be added to hids.h instead of creating a new header file?

Reply:
Ok. I moved all common data structures to hids.h header and include it to hid_c.h. Probablly this is more readable now.

Reply:
Is hids_common.h file still here or am I missing something?